helm/cloud-provider-azure/templates/cloud-provider-azure.yaml

```diff
@@ -430,10 +438,6 @@ spec:
       tolerations:
       - key: CriticalAddonsOnly
         operator: Exists
-      - key: node-role.kubernetes.io/master
```

Comment:
@marosset @jsturtevant can you confirm that this control plane toleration is not needed for the Windows cloud-node-manager pod (i.e., there is no such thing as a "Windows control plane node" in the k8s ecosystem)

Reply:
That's correct, the windows `nodeSelector` above on line 437 is enough to land these on windows.